### DISCUSSION TITLE
How to cross-reference scaffolds/contigs with files in .gfa and .fastg?

### DISCUSSION THREAD

**Discussion by jolespin** [ablab/spades#932] [Q&A]
I'm trying to figure out how to cross-reference the identifiers in the `assembly_graph_with_scaffolds.gfa`, the `assembly_graph.fastg`, and `scaffolds.fasta` files.

**Comment by asl** [comment]
Hello

Scaffolds are represented as paths in the assembly graph. There are already `P` records in GFA for this (note that each record correspond to the scaffold segment as GFA does not allow gapped paths).

The same information is available standalone in `scaffolds.paths` file. Consider familiarising yourself with SPAdes manual (https://cab.spbu.ru/files/release3.15.4/manual.html#spadesoutput) for more information & examples
